src/bunit.web/EventDispatchExtensions/KeyboardEventDispatchExtensions.cs

```diff
@@ -43,6 +43,14 @@ public static void KeyDown(this IElement element, string key, string? code = def
 		/// <param name="eventArgs">The event arguments to pass to the event handler.</param>
 		public static void KeyDown(this IElement element, KeyboardEventArgs eventArgs) => _ = KeyDownAsync(element, eventArgs);
 
+		/// <summary>
+		/// Raises the <c>@onkeydown</c> event on <paramref name="element"/>, passing the provided <paramref name="key"/>
+		/// to the event handler.
+		/// </summary>
+		/// <param name="element">The element to raise the event on.</param>
+		/// <param name="key">The keyboard key to pass to the event handler.</param>
+		public static void KeyDown(this IElement element, Key key) => _ = KeyDownAsync(element, key);
```

Comment:
Compared to this: 

```
public static void KeyDown(this IElement element, string key, string? code = default, float location = default, bool repeat = default, bool ctrlKey = default, bool shiftKey = default, bool altKey = default, bool metaKey = default, string? type = default)
			=> KeyDownAsync(element, new KeyboardEventArgs { Key = key, Code = code!, Location = location, Repeat = repeat, CtrlKey = ctrlKey, ShiftKey = shiftKey, AltKey = altKey, MetaKey = metaKey, Type = type! });
```

There are a few arguments missing I think. Perhaps this is a better overload, that allows folks to do a little extra with the new Key type.

```suggestion
		public static void KeyDown(this IElement element, Key key, bool repeat = default, string? type = default) => ...   
```

What do you think (same for the two other KeyXXXX methods).